File: FusionAndroid/app/src/main/java/fr/intechinfo/fusionandroid/SMS.java
package fr.intechinfo.fusionandroid;


public class SMS {
    private String Address;
    private String Body;
    private String Date;
    private String Type;

    public SMS(String address, String body, String date, String type){
        Address = address;
        Body = body;
        Date = date;
        Type = type;
    }

    public String GetAddress(){
        return Address;
    }
    public String GetBody(){
        return Body;
    }
    public String GetDate(){
        return Date;
    }
    public String GetType(){
        return Type;
    }


}

File: FusionAndroid/app/src/main/java/fr/intechinfo/fusionandroid/SMS.kt
package fr.intechinfo.fusionandroid


class SMS(private val Address: String, private val Body: String, private val Date: String, private val Type: String) {

    fun GetAddress(): String {
        return Address
    }

    fun GetBody(): String {
        return Body
    }

    fun GetDate(): String {
        return Date
    }

    fun GetType(): String {
        return Type
    }


}
